Makefile.doc

```diff
@@ -58,11 +58,19 @@ ifndef QUICK
 SPHINX_DEPS := coq
 endif
 
-sphinx: $(SPHINX_DEPS)
-	$(SHOW)'SPHINXBUILD doc/sphinx'
-	$(HIDE)$(SPHINXENV) $(SPHINXBUILD) -b html $(ALLSPHINXOPTS) doc/sphinx $(SPHINXBUILDDIR)/html
-	@echo
-	@echo "Build finished. The HTML pages are in $(SPHINXBUILDDIR)/html."
+# sphinx-html and sphinx-latex
+sphinx-%: $(SPHINX_DEPS)
+	$(SHOW)'SPHINXBUILD doc/sphinx ($*)'
+	$(HIDE)$(SPHINXENV) $(SPHINXBUILD) -b $* \
+		$(ALLSPHINXOPTS) doc/sphinx $(SPHINXBUILDDIR)/$*
+
+sphinx-pdf: sphinx-latex
+	$(SHOW)'LATEXMK $(SPHINXBUILDDIR)/latex'
+	$(HIDE)+$(MAKE) -C $(SPHINXBUILDDIR)/latex
+
+sphinx:
+	+$(MAKE) sphinx-html
+	+$(MAKE) sphinx-pdf
```

Comment:
The `sphinx-pdf` target should be included in the `doc-pdf` target. There are also quite a few places which mention the `sphinx` target that should be fixed to use `sphinx-html` instead (the other solution would be to keep `sphinx` with its current meaning and add a new `sphinx-all` target).

Reply:
Thanks for pointing this out.  Question: do you know why `install-doc-printable` and `install-doc-sphinx` have no dependencies?

Reply:
I don't. It's probably a mistake.